net/WebSocketHandler.hpp

```diff
@@ -674,7 +674,8 @@ class WebSocketHandler : public ProtocolHandlerInterface
             LOG_WRN("CheckTimeout: Timeout websocket: Ping: last " << _pingMicroS.last() << "us, avg "
                     << _pingMicroS.average() << "us >= " << _pingTimeout.count() << "us over "
                     << (int)_pingMicroS.duration() << "s");
-            shutdownSilent();
+            if( isIPType() )
+                shutdownSilent();
```

Comment:
Outch, missed the internal kit usage obviously, as also not considered in the modified orig code.
Thank you!